pkgs/by-name/li/libfprint-focaltech-2808-a658/package.nix

```diff
@@ -0,0 +1,92 @@
+{
+  stdenv,
+  lib,
+  fetchurl,
+  rpm,
+  cpio,
+  glib,
+  gusb,
+  pixman,
+  libgudev,
+  nss,
+  libfprint,
+  cairo,
+  pkg-config,
+  autoPatchelfHook,
+}:
+
+# https://discourse.nixos.org/t/request-for-libfprint-port-for-2808-a658/55474
+stdenv.mkDerivation rec {
+  pname = "libfprint-focaltech-2808-a658";
+  libso = "libfprint-2.so.2.0.0";
```

Comment:
It'd be cleaner to define variables like this in a let block above the `mkDerivation` call.

Reply:
Moved out `libso` out of derivation. Is that all for this?

Reply:
You may have forgotten to push that change.

Reply:
Done now, let me know if thats okay.